src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -302,12 +325,110 @@ const debug_configuration *debug_configuration::get_instance() {
 #endif
 }
 
-bool debug_configuration::is_dumped_layer(const std::string& layer_name, bool is_output) const {
+bool debug_configuration::is_layer_for_loading(const std::string& layer_name) const {
```

Comment:
could you rename `layer_name` into `id` for uniformity? (or `id` into `layer_name`)

Reply:
Discussed to fix separately